test/BuildTools.Tasks.Tests/RunTaskTests.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.AspNetCore.BuildTools;
+using Xunit;
+
+namespace BuildTools.Tasks.Tests
+{
+    public class RunTaskTests
+    {
+        [Fact]
+        public void ExitCodeIsNonZeroIfFailedToStart()
+        {
+            var engine = new MockEngine { ContinueOnError = true };
+            var task = new Run
+            {
+                FileName = "sdfkjskldfsjdflkajsdas",
```

Comment:
Maybe add a check to ensure this task doesn't exist on disk already?
<br />
<br />
<br />
<br />
<br />
<br />
<br />
:trollface:

Reply:
haha, I was thinking of writing a console tool named sdfkjskldfsjdflkajsdas, but settled on dsfdflkajsdaskjskldfsj instead.